test/NinjaTools.FluentMockServer.API.Exclude.Tests/Downstream/HttpContentEmailData.cs

```diff
@@ -1,22 +1,19 @@
 using System.IO;
 using System.Net.Http;
 using System.Text;
-using Newtonsoft.Json;
+using JetBrains.Annotations;
 using Xunit;
 
 namespace NinjaTools.FluentMockServer.API.Tests.Downstream
 {
     public class HttpContentEmailData : TheoryData<Stream, HttpContent>
     {
         public const int Count = 5;
-        
+
         /// <inheritdoc />
-        public HttpContentEmailData ()
+        public HttpContentEmailData()
         {
-            foreach (var email in RandomEmailFactory.Generate(Count))
-            {
-                AddDataset(email.ToString());
-            }
+            foreach (var email in RandomEmailFactory.Generate(Count)) AddDataset(email.ToString());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)